docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,740 @@
+see :ref:`manual_velocity_control_mechanisms`
+
+Voltage
+=============
+This commands the module to a specific drive voltage. A positive voltage always goes in the counterclockwise direction, and a negative voltage always goes clockwise.
+
+*******************************************
+Updating Firmware with the Control Center
+*******************************************
+Flashing new firmware onto your module is done through the FIRMWARE tab in the Control Center, available on the left side.
+
+All module firmware can be found on your product's page on `Vertiq's website <https://www.vertiq.co/>`_.
+
+Screen Capture Example
+=================================
+The following screen capture demonstrates how to update your module's firmware through the Control Center. In this example, we will be updating a Vertiq 23-06 2200Kv configured to a baud rate of 921600.
+
+.. raw:: html
+
+    <style type="text/css">
+    .center_vid {   margin-left: auto;
+                    margin-right: auto;
+                    display: block;
+                    width: 75%; 
+                }
+    </style>
+    <video class='center_vid' controls><source src="../_static/control_center_pics/firmware_flash_screen_cap.mp4" type="video/mp4"></video>
+
+Firmware Update Instructions
+=================================
+1. Download the file meant for your module from `vertiq.co <https://www.vertiq.co/>`_
+
+.. note::
+    If you downloaded a compressed zip archive from the website, you do not need to extract the contents. Simply use the .zip as your file selection 
+    when selecting your firmware file. It is highly recommended that you use zip archives as they contain important metadata that helps protect you 
+    from flashing incorrect firmware.
+
+2. Navigate to the FIRMWARE tab of the Control Center
+3. Click the Select Firmware button found at the bottom of the page
+
+.. image:: ../_static/control_center_pics/select_firmware.png
+
+4. Use the File Explorer to select the file downloaded from vertiq.co
+5. If you've selected the correct zip archive for your folder, or if you selected a binary file, you will see a Flash button appear. 
+   Click it to begin your firmware update
+
+   .. note::
+        If you are using a zip archive, you will be presented with up to 4 options (see below) depending on the state of your module and the release archive. 
+        Note that you may see as few as 1 option when using a zip archive. The number depends on the type of updates that are deemed to be appropriate for each release:
+           
+           * Flash App: Updates only the application code, which is the section of code that runs during normal operation.
+
+           * Flash Boot: Updates the proprietary bootloader.
+
+           * Flash Upgrade: Updates the proprietary upgrade firmware.
+
+           * Flash Combined: Updates all three sections.
+
+        **If you are unsure of which type of update to perform, it is recommended to select Flash Combined.**
```

Comment:
I might actually say that if they have the app one as an option, hit that, that's the safest. Otherwise, select combined.